docs/source/developing/writing-docs.adoc

```diff
@@ -50,6 +48,51 @@ The latter will start the https://middlemanapp.com[Middleman] server on
 port 4567. You can access the rendered documentation by browsing to
 http://<IP of Docker daemon>:4567.
 
+[[section-deploying-minishift-docs]]
 == Deploying Minishift Documentation
 
```

Comment:
The second level titles seem to be using Title case, whereas the third level ones use Sentence case, not sure if this is the standard we follow from the OpenShift docs.